docs/src/pages/components/icons/icons.md

```diff
@@ -7,61 +7,22 @@ components: Icon, SvgIcon
 
 <p class="description">Guidance and suggestions for using icons with Material-UI.</p>
 
-A [system icon](https://material.io/design/iconography/system-icons.html) or UI icon,
-symbolizes a command, file, device, or directory.
-System icons are also used to represent common actions like trash, print, and save,
-and are commonly found in app bars, toolbars, buttons, and lists.
-Google has provided a set of [Material icons](https://material.io/tools/icons/?style=baseline) that follow these guidelines.
+Material-UI provides icons support in three ways.
 
-Material-UI provides two components to render system icons: `SvgIcon` for rendering SVG paths, and `Icon` for rendering font icons.
+- Standardized Material Design icons are exported as React components from `@material-ui/icons` package (Material Icons).
+- `SvgIcon` component as a React wrapper for custom SVG icon is exported from `@material-ui/core`.
+- `Icon` component as a React wrapper for custom Font icons is exported from `@material-ui/core`.
 
-## SVG Icons
+## Material Icons (recommended)
 
-The `SvgIcon` component takes an SVG `path` element as its child and converts it to a React component that displays the path,
-and allows the icon to be styled and respond to mouse events. SVG elements should be scaled for a 24x24px viewport.
-
-The resulting icon can be used as is,
-or included as a child for other Material-UI components that use icons.
-By default, an Icon will inherit the current text color.
-Optionally, you can set the icon color using one of the theme color properties: `primary`, `secondary`, `action`, `error` & `disabled`.
-
-{{"demo": "pages/components/icons/SvgIcons.js"}}
-
-### SVG Material icons
+Material Design has standardized over 1,000 commonly needed icons. For each icon we export a respective React component from `@material-ui/icons` package. Material Icons are based on SVG icons. You can search the full list of 
```

Comment:
```suggestion
Material Design has standardized over 1,100 commonly needed icons in five different "themes" (see below). For each icon we export the respective React component from  the`@material-ui/icons` package. You can search the full list of 
```

(Arguably many of the provided icons are quite obscure, and some common ones are missing!)

Reply:
I think it is worth noting that our implementation of Material Design icons (that is Material Icons) is SVG based (as it offers benefits over font icon based). What do you think?

I wouldn't put 1100, because over 1000 is more generic. If they implement 1220 in the next year, we'll have to change it to 1200. What do you think?

Reply:
What do you say about: ", each in five different "themes" (see below)."

Reply:
```suggestion
Material Design has standardized over 1,000 commonly needed icons, each in five different "themes" (see below). For each SVG icon we export the respective React component from  the`@material-ui/icons` package. You can search the full list of 
```

Better?

Reply:
Check latest commit please.